packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -409,43 +411,36 @@ $Calendar-dayPicker-margin: 10px;
     }
   }
 
-  // Show the day picker and month picker side-by-side, if enabled
-  .pickerIsMonthPickerVisible {
+  // Without modifying the Pickadate JS, this transparent
+  // button is necessary to toggle the month view.
+  .headerToggleView {
+    height: $Calendar-day;
+    @include ms-left(70px);
+  }
+
+  // Show month picker, if enabled
+  .monthPickerVisible {
     // Swap margin for padding so that the border extends the full height.
     .dayPicker {
       margin: -$Calendar-dayPicker-margin 0;
       padding: $Calendar-dayPicker-margin 0;
     }
+
     // Contains the calendar view for picking a day.
     .dayPicker {
       @include ms-borderBox;
       @include border-right(1px, solid, $ms-color-neutralLight);
-      width: 220px;
+      width: 212px;
       min-height: ( $Calendar-day * 8 ) - $Calendar-dayPicker-margin;
     }
-
-    .holder {
-      width: 440px;
-    }
     // Show the month picker.
     .monthPicker {
       display: block;
     }
-    // Style the month and year pickers.
-    .monthPicker,
-    .yearPicker {
-      top: 9px;
-      @include ms-left(238px);
-      position: absolute;
-    }
-    // Size and position of the month and year labels.
-    .month {
-      @include margin-left(12px);
-    }
 
     .optionGrid {
-      width: 200px;
-      @include margin(10px, 0, 0, 0);
+      height: ( ($Calendar-month + 10) * 3 );
```

Comment:
They are using computations for the day picker
.dayPicker,.monthPicker {
    min-height: ( $Calendar-day * 8 ) - $Calendar-dayPicker-margin;
  }

So that is why I was using a similar function to set the .optionGrid height.